### DISCUSSION TITLE
Login in with Microsoft and flow/login URL

### DISCUSSION THREAD

**Discussion by henrik-funmed** [ory/kratos#1966] [Q&A] [answered]
I am trying to use Ory Kratos and have users login with their existing Microsoft account. After completing the guide at https://www.ory.sh/kratos/docs/concepts/credentials/openid-connect-oidc-oauth2 I cannot get it to work.

My question is what login url the flow should return? If it is left empty, then a page is displayed saying that this configuration should be set:

```
selfservice:
  flows:
    login:
      ui_url: http://my-app.com/login
```

But I want the user to be presented with Microsoft's login form, and then be redirected back when the user is logged in. I don't want to present a custom login form asking for microsoft credentials.

How should the login flow be configured when using Kratos and Microsoft login flow?

**Comment by vinckr** [comment, marked as answer]
Hello @henrik-funmed 
Apologies for the late response. 

Did you try adding Microsofts login form to `ui_url:` ? Can you share an example of this login form ?

**Reply by vinckr** [reply]
Hey @henrik-funmed , just checking in if your problem still persists. 
I will mark this as answered for now, but feel free to come back to the discussion any time.
Thanks a ton :bee:
